pkg/vault/vault.go

```diff
@@ -0,0 +1,176 @@
+package vault
+
+import (
+	"github.com/urfave/cli"
+
+	"fmt"
+	"io/ioutil"
+	"encoding/base64"
+	"encoding/json"
+	"os"
+	"net/url"
+	"net/http"
+	"time"
+
+//	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Using `go imports` will make your life much simpler.

https://godoc.org/golang.org/x/tools/cmd/goimports

You can usually hook it into whatever editor you're using to fire on save.